mailbox/tika/src/test/java/org/apache/james/mailbox/tika/TikaTextExtractorTest.java

```diff
@@ -121,5 +147,38 @@ public void excelTest() throws Exception {
                 "\n" +
                 "\n");
     }
-    
+
+    @Test
+    public void deserializerShouldThrowWhenMoreThanOneNode() throws Exception {
```

Comment:
given the deserialization code complexity, you need much more tests.
Maybe doing unit tests on the deserializer would be a better idea